src/python/zquantum/core/distance_measures.py

```diff
@@ -0,0 +1,108 @@
+### The following code is derived from: https://github.com/GiggleLiu/QuantumCircuitBornMachine/blob/master/LICENSE
+### The original resource is under MIT license, which is pasted below for convenience >>>
+# MIT License
+# Copyright (c) 2017 Leo
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import numpy as np
+
+
+def compute_rbf_kernel(x_i, y_j, sigma):
```

Comment:
These could have typing, though since it's external code I understand if you don't want to modify it.

Reply:
I actually added it and then wasn't sure if it was reasonable, but it's probably worth it since you raised it too. Is there any good reasons why it's better not to change the external code, as far as you know?

Reply:
One reason I can think of is that it might be harder to update it if there are any changes to the original code that we want to introduce or if we want to compare it with the original code.
Another is that if there's a lot of external code, you might accidentally introduce some errors.
But in this case I think it should be ok.

Reply:
Agree. Done it [here](https://github.com/zapatacomputing/z-quantum-core/pull/63/commits/1c3e5168aefda3cdee23839089c41addcfd83c93)